apps/api/.env.example

```
@@ -135,4 +135,33 @@ CDP_API_KEY_SECRET=""
+# - tr (Turkish)          - he (Hebrew)            - th (Thai)
+# - vi (Vietnamese)       - id (Indonesian)        - ms (Malay)
+# - uk (Ukrainian)        - cs (Czech)             - hu (Hungarian)
+#
+# The system automatically excludes URLs containing language indicators
+# for other languages (e.g., /es/, /fr/, ?lang=de, etc.)
+#
+# Examples:
+# DEFAULT_CRAWL_LANGUAGE=en    # English content only
+# DEFAULT_CRAWL_LANGUAGE=es    # Spanish content only  
+# DEFAULT_CRAWL_LANGUAGE=all   # No filtering (crawl all languages)
+DEFAULT_CRAWL_LANGUAGE=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)